BUILD.md

```diff
@@ -6,13 +6,17 @@ Install dependencies:
 
 ```sh
 sudo apt install -y podman dh-python build-essential fakeroot make libqt5gui5 \
-    python3 python3-dev python3-venv python3-pip python3-stdeb python3-all
+    pipx python3 python3-dev python3-stdeb python3-all
 ```
 
-Install poetry (you may need to add `~/.local/bin/` to your `PATH` first):
+Install Poetry using `pipx` (recommended) and add it to your `$PATH`:
+
+_(See also a list of [alternative installation
+methods](https://python-poetry.org/docs/#installation))_
 
 ```sh
-python3 -m pip install poetry
+pipx ensurepath
+pipx install poetry
 ```
 
```

Comment:
Good thinking. I didn't mention it because the `pipx ensurepath` command actually writes this to the user:

https://github.com/pypa/pipx/blob/1c4b644f767b0893291ea71e93757f4c3681a7ea/src/pipx/commands/ensure_path.py#L77-L79

Do you think it's easy to miss?

Reply:
I'd say it is easy to miss. I'd say when following instructions, users only notice when something fails.

Reply:
Ok, I've updated the branch with an instruction for this.